ledger/ledgercore/statedelta.go

```diff
@@ -191,12 +195,10 @@ type AccountDeltas struct {
 // This does not play well for AssetConfig and ApplicationCall transactions on scale
 func MakeStateDelta(hdr *bookkeeping.BlockHeader, prevTimestamp int64, hint int, stateProofNext basics.Round) StateDelta {
 	return StateDelta{
-		Accts:    MakeAccountDeltas(hint),
-		KvMods:   make(map[string]KvValueDelta),
-		Txids:    make(map[transactions.Txid]IncludedTransactions, hint),
-		Txleases: make(map[Txlease]basics.Round),
+		Accts:  MakeAccountDeltas(hint),
+		KvMods: make(map[string]KvValueDelta),
```

Comment:
KvMods seems like a candidate for the same treatment. The vast majority of txns will not have any KvMods.